src/component/legend/LegendView.js

```diff
@@ -238,7 +238,7 @@ export default echarts.extendComponentView({
                         );
 
                         // FIXME: consider different series has items with the same name.
-                        itemGroup.on('click', curry(dispatchSelectAction, name, api))
+                        itemGroup.on('click', curry(dispatchSelectAction, name, null, api, excludeSeriesId))
```

Comment:
The name here is "item name", but in `dispatchHighlightAction` and `dispatchDownplayAction` the first parameter is `seriesName`.

Reply:
The name here is `seriesName` or `itemName`. It depends on series.